apps/frontend/src/libs/components/app-header/app-header.tsx

```diff
@@ -69,6 +78,16 @@ const AppHeader: React.FC = () => {
 						<img alt="Open menu" className={arrowClassName} src={arrowDown} />
 					</button>
 				</div>
+				<button
+					aria-label="Open user menu"
+					className={burgerMenuClassName}
+					onClick={handleMenuToggle}
+					type="button"
+				>
+					<div />
+					<div />
+					<div />
```

Comment:
Do you need these?

Reply:
Please, add at least class names for them so that the purpose of these divs can be understood easily